Client/mods/deathmatch/logic/CClientDFF.h

```diff
@@ -46,9 +46,13 @@ class CClientDFF : public CClientEntity
     static bool IsDFFData(const SString& strData);
 
     // Sorta a hack that these are required by CClientEntity...
-    void Unlink(void){};
-    void GetPosition(CVector& vecPosition) const {};
-    void SetPosition(const CVector& vecPosition){};
+
+    void     Unlink(void){};
+    void     GetPosition(CVector& vecPosition) const {};
+    void     SetPosition(const CVector& vecPosition){};
+    RpClump* GetLoadedClump(ushort usModelId);
+    ushort   m_usModelID;
+    SString  m_strModelName;
```

Comment:
Please don't expose class member variables. Use either protected or private scope for these.